src/compas/robots/model/link.py

```diff
@@ -4,7 +4,7 @@
 
 from compas.files import URDF
 from compas.files import URDFParser
-from compas.geometry.xforms import Transformation
+from compas.geometry.transformations import Transformation
```

Comment:
the other packages in compas should always use the "second level import" convention to allow package developers to move stuff around without affecting the functionality of other packages.

this should have been `from compas.geomtry import Transformation` from the beginning...

Reply:
got it, updated